### PR TITLE
draft AI policy

### PR DESCRIPTION
**Pull request by denis-yuen**
**Description**
In plain English, explain our policy on generative AI/LLMs for users. 

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-6560

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If the PR has any new pages, only **AFTER** the PR is approved by all reviewers, generate a new discourse topic, by running the action at https://github.com/dockstore/dockstore-documentation/actions/workflows/add-discourse-topic.yml. Otherwise we may end up with orphaned topics, which would need manual management.

### CODE REVIEW

**Comment by denis-yuen** [comment]
Merge with https://github.com/dockstore/dockstore-documentation/pull/287, I need the link fixes in that

File: docs/faq.rst [hunk 1]
```diff
@@ -84,6 +84,27 @@ When viewing an individual tool or workflow, you can tell which versions are Ope
...
+-----------------------------------------------------------------------
+
+In broad strokes, we are exploring the use of generative AI to enhance the workflow and tool development process.
+For full transparency, we will work following these principles:
+
+1. For content generated and displayed to the public or to other users (e.g. AI generated topic sentences), we will only train or use as prompts, data that is already available publicly.
+In other words, as an example, only after the publish button is used, will we create topic sentences that are presented to other users.
+2. For content displayed to the users themselves (e.g. auto-complete or auto-categorization type features), we only use in-house LLMs or AIs. In other words, we will only
+utilize LLMs hosted at OICR or UCSC, ensuring that no private data leaves the Dockstore ecosystem.
+3. AI generated will be clearly labelled, except in the case that a workflow developer, curator, or similar human has "approved" of it
```
Comment (svonworl) [inline docs/faq.rst:99]: "AI generated" -> "AI-generated content"?

File: docs/faq.rst [hunk 2]
```diff
@@ -84,6 +84,27 @@ When viewing an individual tool or workflow, you can tell which versions are Ope
 
 :ref:`(back to top) <topFAQ>`
 
+.. _what-is-dockstore-generative-ai-policy:
+
+In human readable terms, what is Dockstore's approach to generative AI?
```
Comment (svonworl) [inline docs/faq.rst:89]: IMHO, we can remove "In human readable terms,".

File: docs/faq.rst [hunk 3]
```diff
@@ -84,6 +84,27 @@ When viewing an individual tool or workflow, you can tell which versions are Ope
...
+-----------------------------------------------------------------------
+
+In broad strokes, we are exploring the use of generative AI to enhance the workflow and tool development process.
+For full transparency, we will work following these principles:
+
+1. For content generated and displayed to the public or to other users (e.g. AI generated topic sentences), we will only train or use as prompts, data that is already available publicly.
+In other words, as an example, only after the publish button is used, will we create topic sentences that are presented to other users.
```
Comment (svonworl) [inline docs/faq.rst:96]: Regarding in-house AI usage, the deciding factor is whether the source data is public or private.  If we reword in those terms, we cover the scenario where we use AI on public data to generate information that we only display privately.  There's probably also a theoretical scenario where we could use AI on private data (to summarize it?) and generate something that was safe to display to the public...

File: docs/faq.rst [hunk 4]
```diff
@@ -84,6 +84,27 @@ When viewing an individual tool or workflow, you can tell which versions are Ope
...
+2. For content displayed to the users themselves (e.g. auto-complete or auto-categorization type features), we only use in-house LLMs or AIs. In other words, we will only
+utilize LLMs hosted at OICR or UCSC, ensuring that no private data leaves the Dockstore ecosystem.
+3. AI generated will be clearly labelled, except in the case that a workflow developer, curator, or similar human has "approved" of it
+4. We will attempt to programmatically review generated content but cannot guarantee that we will successfully review everything. We are still very much exploring what these technologies
```
Comment (david4096) [inline docs/faq.rst:100]: Missing a period here and next number.

File: docs/faq.rst (hunk 2 above)
Comment (denis-yuen) [reply docs/faq.rst:89]: :shrug:

File: docs/faq.rst (hunk 3 above)
Comment (denis-yuen) [reply docs/faq.rst:96]: :thought_balloon:

File: docs/faq.rst [hunk 5]
```diff
@@ -84,6 +84,28 @@ When viewing an individual tool or workflow, you can tell which versions are Ope
...
+.. _what-is-dockstore-generative-ai-policy:
+
+In plain language, what is Dockstore's approach to generative AI?
+-----------------------------------------------------------------------
+
+In broad strokes, we are exploring the use of generative AI to enhance the workflow and tool development process.
+For full transparency, we will work following these principles:
+
+1. For content generated and displayed to the public or to other users (e.g. AI generated topic sentences), we will only train or use as prompts, data that is already available publicly.
+In other words, as an example, only after the publish button is used, will we create topic sentences that are presented to other users.
```
Comment (coverbeck) [inline docs/faq.rst:96]: Nit: instead of `after the publish button is used`, I'd suggest something like `after publishing a tool  or workflow`, to handle non-UI published entries. API publishing probably isn't that common, but .dockstore.yml publishing is.

**Review by coverbeck — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Review by david4096 — APPROVED** [review]

**Comment by aofarrel** [comment]
I'm late to the party, but this gets a ✔️ from me!
